scripts/source-maintenance.sh

```diff
@@ -62,8 +69,8 @@ else
 fi
 
 ASVER=`astyle --version 2>&1 | grep -o -E "[0-9.]+"`
-if test "${ASVER}" != "2.04" ; then
-	echo "Astyle version problem. You have ${ASVER} instead of 2.04"
+if test "${ASVER}" != "${TargetAstyleVersion}" ; then
+	echo "Astyle version problem. You have ${ASVER} instead of ${TargetAstyleVersion}"
```

Comment:
When the version was specified explicitly, this should become a fatal error, not a warning. The user insisted that we use astyle version X to format sources. Proceeding without any formatting is clearly not what the user wanted!

Reply:
I disagree. If the user is forcing a specific binary, they are expressing a clear intent and we should be trusting them to know what they're doing. For instance, they may be testing a different atyle version.
Let's also keep in perspective that "the user" is us, most likely in the very specific context of automated source formatting

Reply:
> we should be trusting them to know what they're doing

I agree. I should have spelled out "this". I did not, leading to a misunderstanding. It does not matter now because the (real) problem I was documenting has been resolved by switching from "version" to "executable" configuration.